pkg/sdkserver/sdkserver.go

```diff
@@ -275,21 +273,41 @@ func (s *SDKServer) updateState() error {
 
 	s.gsUpdateMutex.RLock()
 	gs.Status.State = s.gsState
+
+	// if we are setting the Reserved status, check for the duration, and set that too
```

Comment:
nit: Comments should be proper sentences, starting with a capital letter and ending with a period. 

https://github.com/golang/go/wiki/CodeReviewComments#comment-sentences